sys/linux/dev_img_rogue.txt

```diff
@@ -0,0 +1,1925 @@
+} [packed]
+
+PVRSRV_BRIDGE_OUT_PMRUNREFUNLOCKPMR {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_PVRSRVUPDATEOOMSTATS {
+	ui32ui32StatType	int32
+	ui32pid			int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_PVRSRVUPDATEOOMSTATS {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_TLACQUIREDATA {
+	hSD	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLACQUIREDATA {
+	ui32ReadOffset	int32
+	ui32ReadLen	int32
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_TLCLOSESTREAM {
+	hSD	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLCLOSESTREAM {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_TLCOMMITSTREAM {
+	hSD		IMG_HANDLE
+	ui32ReqSize	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLCOMMITSTREAM {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_TLDISCOVERSTREAMS {
+	puiNamePattern	ptr[in, array[int8]]
+	ui32Size	len[puiStreams, int32]
+	puiStreams	ptr[in, array[int8]]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLDISCOVERSTREAMS {
+	puiStreams	ptr[out, array[int8]]
+	ui32NumFound	int32
+	eError		int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_TLOPENSTREAM {
+	puiName		ptr[in, array[int8]]
+	ui32Mode	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLOPENSTREAM {
+	hSD	IMG_HANDLE
+	hTLPMR	IMG_HANDLE
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_TLRELEASEDATA {
+	hSD		IMG_HANDLE
+	ui32ReadOffset	int32
+	ui32ReadLen	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLRELEASEDATA {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_TLRESERVESTREAM {
+	hSD		IMG_HANDLE
+	ui32Size	int32
+	ui32SizeMin	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLRESERVESTREAM {
+	ui32BufferOffset	int32
+	ui32Available		int32
+	eError			int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_TLWRITEDATA {
+	hSD		IMG_HANDLE
+	ui32Size	len[psData, int32]
+	psData		ptr[in, array[int8]]
+} [packed]
+
+PVRSRV_BRIDGE_OUT_TLWRITEDATA {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCLEARBREAKPOINT {
+	hPrivData	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXCLEARBREAKPOINT {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXDISABLEBREAKPOINT {
+	hPrivData	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXDISABLEBREAKPOINT {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXENABLEBREAKPOINT {
+	hPrivData	IMG_HANDLE
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXENABLEBREAKPOINT {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXOVERALLOCATEBPREGISTERS {
+	ui32TempRegs	int32
+	ui32SharedRegs	int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXOVERALLOCATEBPREGISTERS {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXSETBREAKPOINT {
+	hPrivData		IMG_HANDLE
+	eFWDataMaster		int32
+	ui32BreakpointAddr	int32
+	ui32HandlerAddr		int32
+	ui32DM			int32
+} [packed]
+
+PVRSRV_BRIDGE_OUT_RGXSETBREAKPOINT {
+	eError	int32
+} [packed]
+
+PVRSRV_BRIDGE_IN_RGXCREATECOMPUTECONTEXT {
+	ui32Priority				int32
+	ui32FrameworkCmdize			len[psFrameworkCmd, int32]
+	psFrameworkCmd				ptr[in, array[int8]]
+	hPrivData				IMG_HANDLE
+	ui32StaticComputeContextStateSize	len[psStaticComputeContextState, int32]
+	psStaticComputeContextState		ptr[in, array[int8]]
+	ui32PackedCCBSizeU88			int32
+	ui32ContextFlags			int32
```

Comment:
It looks like RGX_CONTEXT_FLAG_DISABLESLR is a possible flag value for this field.

Reply:
Fixed.